### PR TITLE
Local nav data

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/PremiumPageHeader/PremiumPageHeaderController.class.php [hunk 1]
```diff
@@ -17,7 +17,37 @@ public function wikiHeader() {
...
+			'Videos',
+			'Images',
+		];
+
+		return [
+			'text' => 'Explore',
+			'children' => array_map( function ( $page ) {
+				$title = Title::newFromText( $page, NS_SPECIAL );
+				return [
+					'text' => $title->getText(),
```
Comment (latata) [inline extensions/wikia/PremiumPageHeader/PremiumPageHeaderController.class.php:40]: do we need both `text` and `textEscaped`?

File: extensions/wikia/PremiumPageHeader/scripts/banner-notifications-hack.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+require(['wikia.window', 'jquery'], function (window, $) {
+	'use strict';
+
+	function moveBannerNotifications() {
+		var $globalNav = $('.wds-global-navigation-wrapper'),
+			$notificationsWrapper = $('.banner-notifications-wrapper');
+
+		$notificationsWrapper.detach();
```
Comment (weronika) [inline extensions/wikia/PremiumPageHeader/scripts/banner-notifications-hack.js:8]: detach is redundant I guess

File: extensions/wikia/PremiumPageHeader/scripts/banner-notifications-hack.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+		$notificationsWrapper.insertAfter($globalNav);
+
+		window.BannerNotification.prototype.onShow = window.BannerNotification.prototype.show;
+
+		window.BannerNotification.prototype.show = function () {
+			this.onShow();
+
+			var $notificationsWrapper = $('.banner-notifications-wrapper');
+
+			$notificationsWrapper.detach();
```
Comment (weronika) [inline extensions/wikia/PremiumPageHeader/scripts/banner-notifications-hack.js:18]: the same here - redundant detach

File: extensions/wikia/PremiumPageHeader/scripts/banner-notifications-hack.js (hunk 2 above)
Comment (latata) [inline extensions/wikia/PremiumPageHeader/scripts/banner-notifications-hack.js:8]: I think we don't need detach here - insertAfter detaches element.

File: extensions/wikia/PremiumPageHeader/PremiumPageHeaderController.class.php [hunk 4]
```diff
@@ -17,7 +17,37 @@ public function wikiHeader() {
 	}
 
 	public function navigation() {
-		$this->setVal( 'data', ( new NavigationModel() )
-			->getLocalNavigationTree( NavigationModel::WIKI_LOCAL_MESSAGE ) );
+		$this->setVal( 'data', ( new NavigationModel() )->getLocalNavigationTree( NavigationModel::WIKI_LOCAL_MESSAGE ) );
```
Comment (weronika) [inline extensions/wikia/PremiumPageHeader/PremiumPageHeaderController.class.php]: let's use more informative var name, like navTree or navItems
